src/use-lunatic/commons/variables/lunatic-variables-store.ts

```diff
@@ -33,6 +33,8 @@ export type EventArgs = {
 		value: unknown;
 		/** Iteration changed (for array). */
 		iteration?: IterationLevel | undefined;
+		/** removedIndex: when resize an array directly with only one handleChange (remove one line in tableLoop) */
```

Comment:
- No need to put `"removeIndex"` in comment again.
- Documentation comments (i.e. between `/** */`) should start with uppercase and end with a comma.

The comment is not very clear though.